### PR TITLE
Reassign copyright from Digital Mars to D Language Foundation

### PR DESCRIPTION
**Pull request by wilzbach**
https://github.com/dlang/dmd/pull/6771#discussion_r127611531

> CC @WalterBright @andralex - can we replace/update the copyright notice from Digital Mars to D Language Foundation?
> I'm okay with it. @WalterBright ?

### CODE REVIEW

**Comment by JinShil** [comment]
@wilzbach Are you still interested in pursuing this?  I'm tempted to close it.  I'm not sure why noone responded to it.  Maybe it was too big to review.  Could you maybe split in into pieces (e.g. frontend and backend, or some other) to make it easier to review?

**Comment by ibuclaw** [comment]
@wilzbach - I hope you didn't do this by hand...

**Comment by ibuclaw** [comment]
@JinShil - This is a trivial patch, don't close.

@WalterBright - Ping.  Are we good to do this?  Anything legal to take care of?

**Comment by wilzbach** [comment]
>  I hope you didn't do this by hand...

Of course not. Simple replacement over all file headers.

> Are you still interested in pursuing this?

Absolutely, but as long as we don't have Walter's approval, it doesn't make sense to rebase this PR or switch to @ibuclaw's tool.

**Comment by WalterBright** [comment]
No point bothering with this until we get an answer from gcc.

**Review by WalterBright — APPROVED** [review]

**Comment by andralex** [comment]
@wilzbach please rebase

**Review by andralex — APPROVED** [review]

**Comment by wilzbach** [comment]
> @wilzbach please rebase

Rebased & should be ready to go now :)

File: src/ddmd/root/response.d [hunk 1]
```diff
@@ -39,7 +39,7 @@ import ddmd.root.filename;
  * argc and argv are adjusted to be the new command line arguments
  * after response file expansion.
  *
- * Digital Mars's MAKE program can be notified that a program can accept
+ * D Language Foundation's MAKE program can be notified that a program can accept
```
Comment (JinShil) [inline src/ddmd/root/response.d]: I think the MAKE program is still property of Digital Mars.

**Review by JinShil — CHANGES_REQUESTED** [review]

File: docs/man/man1/dmd.1 [hunk 2]
```diff
@@ -1,4 +1,4 @@
-.TH DMD 1 "2011-02-22" "Digital Mars" "Digital Mars D"
+.TH DMD 1 "2011-02-22" "D Language Foundation" "D Language Foundation"
```
Comment (JinShil) [inline docs/man/man1/dmd.1]: I think the precise wording should be "the D Language Foundation".  See the bottom of http://dlang.org.

**Review by JinShil — CHANGES_REQUESTED** [review]

File: docs/man/man1/dmd.1 (hunk 2 above)
Comment (andralex) [reply docs/man/man1/dmd.1]: Correct, our brand includes the "The".

Comment (wilzbach) [reply docs/man/man1/dmd.1]: Ok. Should I just replace it for the man page or for all occurrences?

Comment (ZombineDev) [reply docs/man/man1/dmd.1]: For considering, I say all.

Comment (JinShil) [reply docs/man/man1/dmd.1]: Yes, all.

**Review by JinShil — APPROVED** [review]

**Comment by wilzbach** [comment]
>> Ok. Should I just replace it for the man page or for all occurrences?
> For consistency, I say all.

Ok. Done

**Pull request merged**